pkg/cloud_provider/lustre/lustre.go

```diff
@@ -273,6 +275,41 @@ func (sm *lustreServiceManager) GetCreateInstanceOp(ctx context.Context, instanc
 	}
 }
 
+func (sm *lustreServiceManager) UpdateInstance(ctx context.Context, instance *ServiceInstance) (*ServiceInstance, error) {
```

Comment:
Would `ResizeInstance` be a more appropriate function name?

Reply:
the underlying lustre client calls UpdateInstance API, so the name seems reasonable to me

Reply:
I'm thinking from an extensibility perspective. Right now, this function only updates the instance capacity, but in the future we may also want to support updating other instance parameters, such as the perf tier, file striping, etc. (None of them are coming in S2 though.)